templates/default/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM openliberty/open-liberty:full-java8-openj9-ubi
```

Comment:
can we just use the built-in MST Liberty Dockerfile ?  

This way we don't collide with our own "outer loop" Dockerfile.